docs/images/edge/01_OH_overview.svg

```diff
@@ -22,7 +22,7 @@
 	.st16{font-size:12px;}
 	.st17{fill:none;}
 </style>
-<title>01_OH_overview</title>
+<title>01_IEAM_overview</title>
```

Comment:
Did we want the title to be "01_IEAM_overview"?

Reply:
removed the title